### PR TITLE
adding total issuance

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: scripts/tokenomics-emulation/config.json [hunk 1]
```diff
@@ -111,6 +111,9 @@
         "name": "cere-user-to-app",
         "amount": 1,
         "token_value": 1
+      },
+      {
+        "name": "total-issuance"
```
Comment (AndreiNavoichyk) [inline scripts/tokenomics-emulation/config.json:116]: What are you going to do with `total-issuance`? Update, Increase? Get?

File: scripts/tokenomics-emulation/emulations/total-issuance.emulation.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { IEmulation } from "./emulation.interface";
+import Network from "../network";
+
+class TotalIssuanceEmulation implements IEmulation {
```
Comment (AndreiNavoichyk) [inline scripts/tokenomics-emulation/emulations/total-issuance.emulation.ts:4]: Fix naming.

**Review by AndreiNavoichyk — APPROVED** [review]

File: scripts/tokenomics-emulation/config.json (hunk 1 above)
Comment (karthikeyan-cere) [reply scripts/tokenomics-emulation/config.json]: Renamed as `fetch-total-issuance`

File: scripts/tokenomics-emulation/emulations/total-issuance.emulation.ts (hunk 2 above)
Comment (karthikeyan-cere) [reply scripts/tokenomics-emulation/emulations/total-issuance.emulation.ts]: Fixed

**Pull request merged**